GalobawiS.rtf

```diff
@@ -0,0 +1,15 @@
+{\rtf1\ansi\ansicpg1252\cocoartf2512
+\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
+{\colortbl;\red255\green255\blue255;}
+{\*\expandedcolortbl;;}
+\margl1440\margr1440\vieww10800\viewh8400\viewkind0
+\pard\tx720\tx1440\tx2160\tx2880\tx3600\tx4320\tx5040\tx5760\tx6480\tx7200\tx7920\tx8640\pardirnatural\partightenfactor0
+
+\f0\fs24 \cf0 Hi my name is Galobawi Salih\
+\
+Three words that describe me are:\
+ 1. Genuine\
+2. Kind\
+3. Humorous\
+\
+My Favorite things to do during my free time is to be active. I like to go hiking and bike riding. If the weather is not nice out, then I enjoy spending my time inside playing my PS4. My favorite games on the PS4 is Uncharted 4.}
```

Comment:
Do you ever play Rocket League? I met my wife on Rocket League about 4 years ago. I'm not a big gamer (my wife was/is more of a gamer than me), but I play Overwatch every once in a while now.